main.js

```diff
@@ -266,7 +267,7 @@ app = {
 			url: url
 		}).done((data) => {
 			const word = String(data["ParsedResults"][0]["ParsedText"]).trim().toLowerCase().split(" ").join("");
-			if (word.length > 3) {
+			if (word.length > 2) {
```

Comment:
cause com is not more than 3 
cause val is not more than 3
cause xml is not more than 3 
etc .....